### PR TITLE
#180422070

### PR DESCRIPTION
**Pull request by SalShai**
## Feature description
Modify the Payroll entry according to the Bank template

## Solution description
- Customizing `Employee Details` Child Table in payroll Entry Doc. 
- Fetching Employee Details from the Employee Doc.
- Fetching Bank Details of the employee from the Bank Account List.
- Updating the payment amount of the employee, but fetching NetPay from the Salary Slip once salary slip is submitted.

## Output screenshots (optional)
Fetching up of Bank Details when `Get Employees` Button is clicked
Screen Shot 2021-11-25 at 12 23 57 PM

Fetching Payment Amount(Net Pay) on the `Create Salary Slip` Button is clicked.
Screen Shot 2021-11-28 at 10 11 47 AM

## Areas affected and ensured
- Employee Details Table during `Fill Employee Details`
- Fetching of Salary Slip Net Pay for each employee in the table during `Create Salary Slip` 

## Is there any existing behavior change of other features due to this code change?
Yes, Employee Details fetches more data than before. It needs Bank Account for Each employee in the Table.

## Was this feature tested on all the browsers?
  - [x] Chrome
  - [] Safari

### CODE REVIEW

File: one_fm/api/doc_methods/payroll_entry.py [hunk 1]
```diff
@@ -30,6 +28,64 @@ def get_count_holidays_of_employee(self, employee):
...
+
+	Returns:
+		list of active employees based on selected criteria
+		and for which salary structure exists.
+	"""	
+	self.set('employees', [])
+	employees = self.get_emp_list()
+	
+	#Fetch Bank Details and update employee list 
+	get_bank_details(employees)
```
Comment (Ehtesham599) [inline one_fm/api/doc_methods/payroll_entry.py]: Go with proper naming for this method as this method is setting the bank details under the employee details and not just getting. `set_bank_details` would be more appropriate.

**Review by Ehtesham599 — CHANGES_REQUESTED** [review]
@SalShai just a small change requested.
Everything else LGTM!

File: one_fm/api/doc_methods/payroll_entry.py (hunk 1 above)
Comment (SalShai) [reply one_fm/api/doc_methods/payroll_entry.py]: Okay. Sure.👍🏼

**Comment by SalShai** [comment]
@Ehtesham599 Made the required changes. Kindly, have a look.

**Review by Ehtesham599 — APPROVED** [review]

**Pull request merged**